tests/acl/test_acl.py

```diff
@@ -538,7 +538,10 @@ def create_or_remove_acl_table(duthost, acl_table_config, setup, op, topo):
 
 
 @pytest.fixture(scope="module")
-def acl_table(duthosts, rand_one_dut_hostname, setup, stage, ip_version, tbinfo):
+def acl_table(duthosts, rand_one_dut_hostname, setup, stage, ip_version, tbinfo,
+              # make sure the tear down of core_dump_and_config_check happened after acl_table
+              core_dump_and_config_check
+              ):
```

Comment:
I'm not sure how many times will the fixture core_dump_and_config_check execute at this sistuation. Maybe twice?

Reply:
It will be only execute once, the import here only help to explicitly decide the execution order.
https://elastictest.org/scheduler/testplan/66d506cbbd14ce56b2e8292d?testcase=acl%2Ftest_acl.py&type=console
![image](https://github.com/user-attachments/assets/0cc591c0-5ab2-4af8-bf05-e9d98b4a678e)
![image](https://github.com/user-attachments/assets/6bb5049d-1b20-4bac-84f6-268905ab20de)

Reply:
And the reason I need to explicitly set the order, is the tear_down of acl_table happens after the tear_down of core_dump_and_config_check, even though the setup of core_dump_and_config_check happens before acl_table.
This order issue should be a bug/issue from pytest.
The tear down of acl_table teardown relies on some test configuration, but they got removed by the tear down of core_dump_and_config_check.